unsupported.html

```diff
@@ -16,10 +16,9 @@
 </head>
 
 <body>
-    <h1>Unsupported Browser</h1>
-    <p>Sorry, the browser you're currently using is out of date and
-        incompatible with geojson.io. Please upgrade
+    <h1>Unsupported Browser or missing API</h1>
+    <p>Sorry, the browser you're currently using do not seems to support WebGL. It could be disabled, or it could be because it is out of date. Please check if WebGL is supported and enabled, or supported, or upgrade
```

Comment:
A couple small suggested edits:

```suggestion
    <p>Sorry, the browser you're currently using does not seems to support WebGL. It could be that WebGL is disabled, or it could be that the browser is out of date. Please check if WebGL is supported and enabled, or supported, or upgrade
```